core/src/main/java/io/questdb/cutlass/line/AbstractLineSender.java

```diff
@@ -136,6 +137,11 @@ public AbstractLineSender field(CharSequence name, boolean value) {
         return this;
     }
 
+    public AbstractLineSender field(CharSequence name, Instant value) {
+        field(name).put(value.toEpochMilli()).put('t');
```

Comment:
you are converting into milliseconds, but it should be in micros. 
see: https://questdb.io/docs/reference/api/ilp/columnset-types/#timestamp

Reply:
You are absolutely right, fixed in the last commit.